docs/advanced-topics/docker-alternatives.rst

```diff
@@ -16,7 +16,7 @@ environment.
 
```

Comment:
The paragraph that starts at line 7 doesn't mention podman, but it does mention singularity and rootless Docker. I would consider removing "including Singularity and rootless Docker" from that paragraph, and adding a very small table of contents under the paragraph that lists the alternatives we talk about (so people can quickly jump to the singularity section, the podman section, or the rootless Docker section).